requirements.txt

```diff
@@ -5,3 +5,4 @@ pytest==8.3.4
 pytest-asyncio==0.24.0
 pytest-cov==5.0.0
 pytest-mock==3.14.0
+trustme==1.2.1
```

Comment:
Latest version of `trustme` is not available for Python 3.8. Here is a PR to drop Python 3.8, as it's EOL: https://github.com/aio-libs/aiosmtpd/pull/555